src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -2179,12 +2180,10 @@ private function step_in_table(): bool {
 					 * >
 					 * > Otherwise, insert the characters given by the pending table
 					 * > character tokens list.
-					 * > …
-					 * > ASCII whitespace is U+0009 TAB, U+000A LF, U+000C FF, U+000D CR, or U+0020 SPACE.
 					 *
 					 * @see https://html.spec.whatwg.org/#parsing-main-intabletext
 					 */
-					if ( strlen( $text ) === strspn( $text, "\t\n\f\r " ) ) {
+					if ( strlen( $text ) === strspn( $text, " \t\f\r\n" ) ) {
```

Comment:
Updated these @sirreal purely to harmonize with the existing checks, which I think are universally in this order, plus whitespace is defined in the spec and used somewhat ubiquitously around here. I don't think it's necessary to repeat here.